frontend/app/components/api/api-v3/hal-resource-dms/help-text-dm.service.ts

```diff
@@ -38,11 +38,11 @@ export class HelpTextDmService {
   }
 
   public loadAll():ng.IPromise<CollectionResourceInterface> {
-    return this.halRequest.get(this.v3Path.help_texts());
+    return this.halRequest.get(this.v3Path.help_texts()) as ng.IPromise<CollectionResourceInterface>;
```

Comment:
Which type does `this.halRequest.get(this.v3Path.help_texts())`return? If its any, the cast should not be necessary.

(this comment also applies to several other places below)